lib/ae_mdw/db/model.ex

```diff
@@ -528,6 +528,19 @@ defmodule AeMdw.Db.Model do
   @type aexn_contract_symbol() ::
           record(:aexn_contract_symbol, index: aexn_contract_symbol_index())
 
+  # AEX-N meta info sorted by txi_idx:
+  #     index: {type, {txi, idx}, pubkey}
+  #     unused: nil
+  @aexn_contract_creation_defaults [
+    index: {nil, {-1, -1}, nil},
+    unused: nil
+  ]
+  defrecord :aexn_contract_creation, @aexn_contract_creation_defaults
+
+  @type aexn_contract_creation_index() :: {aexn_type(), txi_idx(), pubkey()}
```

Comment:
It it possible for the same event (local index) to create two different contracts? Otherwise we can move pubkey out of the index and reduce the index size

Reply:
Done